source/mir/random/ndvariable.d

```diff
@@ -279,6 +280,126 @@ nothrow @safe version(mir_random_test) unittest
     assert(fabs(x[0] + x[1] + x[2] - 1) < 1e-10);
 }
 
+/++
+Multinomial distribution.
++/
+struct MultinomialVariable(U:size_t, T)
```

Comment:
`struct MultinomialVariable(T)`

Reply:
It should be strictly `uint`, we shouldn't use `size_t` here at all

Reply:
Done